website/docs/reference/configs-and-properties.md

```diff
@@ -21,6 +21,21 @@ Whereas you can use **configurations** to:
 * Declare whether a resource should persist its descriptions as comments in the database
 * Apply tags and "meta" properties
 
+### Resource, properties, and configs table
+
+| Resource type | Description      | Supported properties  | Supported configurations    | Config inheritance   |
+|---------------|------------------|-----------------------|-----------------------------|----------------------|
+| Models        | Describes models and their columns | columns, tests, docs, macros   | tags, materialized, persist_docs, tags, etc. | `properties.yml` -> `.yml file` -> `dbt_project.yml`     |
+| Snapshots     |   Periodic snapshots of data              | tests, macros       | tags, materialized, etc.    |   In-file `config()` block -> `.yml file` -> `dbt_project.yml`            |
+| Seeds         |  Uploaded source data for models         | tests, macros            | tags, persist_docs, etc.     |    In-file `config()` block -> `.yml file` -> `dbt_project.yml`              |
+| Tests         | Asserts "truths" about models    | description, tags, macros       |     In-file `config()` block -> `.yml file` -> `dbt_project.yml`         |
```

Comment:
Is there content missing from the "Supported properties" or "Supported configurations" columns for the "Tests" row?

If there isn't supposed to be anything in the "Supported configurations" column, then:
```suggestion
| Tests         | Asserts "truths" about models    | description, tags, macros       | |     In-file `config()` block -> `.yml file` -> `dbt_project.yml`         |
```